translations/monero-core_es.ts

```diff
@@ -221,1153 +221,1161 @@
-        <translation type="unfinished"></translation>
+        <translation>Gastos re-escaneados correctamente</translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="599"/>
         <location filename="../pages/Transfer.qml" line="660"/>
         <source>Please choose a file</source>
-        <translation type="unfinished"></translation>
+        <translation>Escoja un fichero</translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="611"/>
         <source>Can&apos;t load unsigned transaction: </source>
-        <translation type="unfinished"></translation>
+        <translation>No se puede cargar la transacción no firmada:</translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="618"/>
         <source>
 Number of transactions: </source>
-        <translation type="unfinished"></translation>
+        <translation>
+Número de transacciones: </translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="620"/>
         <source>
 Transaction #%1</source>
-        <translation type="unfinished"></translation>
+        <translation>
+Transacción #%1</translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="621"/>
         <source>
 Recipient: </source>
-        <translation type="unfinished"></translation>
+        <translation>
+Beneficiario: </translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="622"/>
         <source>
 payment ID: </source>
-        <translation type="unfinished"></translation>
+        <translation>
+ID de pago: </translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="623"/>
         <source>
 Amount: </source>
-        <translation type="unfinished"></translation>
+        <translation>
+Cantidad: </translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="624"/>
         <source>
 Fee: </source>
-        <translation type="unfinished"></translation>
+        <translation>
+Comisión:</translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="625"/>
         <source>
 Ringsize: </source>
-        <translation type="unfinished"></translation>
+        <translation>
+Tamaño de ring: </translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="639"/>
         <source>Confirmation</source>
-        <translation type="unfinished"></translation>
+        <translation>Confirmación</translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="667"/>
         <source>Can&apos;t submit transaction: </source>
-        <translation type="unfinished"></translation>
+        <translation>No se puede presentar la transacción:</translation>
```

Comment:
I needed to differentiate between send and commit, but I agree that **enviar** sound better here.